gym_sts/rl/mvp.py

```diff
@@ -81,12 +81,24 @@ def check_rllib_bug(space: spaces.Space):
     use_gpu=ff.Boolean(False),
 )
 
+SINGLE_COMBAT = ff.DEFINE_dict(
+    "single_combat",
+    use=ff.Boolean(False),
+    ascension=ff.Integer(20),
```

Comment:
`ascension` could be part of the `ENV` dict, it's not specific to `SINGLE_COMBAT`.

Reply:
Done